src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx

```diff
@@ -124,7 +123,10 @@ Below is a table comparing what basic users and full users can do. In short: bas
 
         <tr>
           <td>
-            Digital Experience Monitoring UI (browser, mobile, synthetics)
+            Digital Experience Monitoring UI, which includes: 
```

Comment:
@zuluecho9, maybe in a future ticket, consider moving these sections into separate tables? In dark mode, these sub-headings are difficult to read and might be clearer if they were in their own tables.